nbqa/cmdline.py

```diff
@@ -66,8 +66,8 @@ def __str__(self) -> str:
 
         return " ".join(args)
 
-    @staticmethod
-    def parse_args(raw_args: Optional[List[str]]) -> "CLIArgs":
+    @classmethod
+    def parse_args(cls, raw_args: Optional[List[str]]) -> "CLIArgs":
```

Comment:
Any reason for making it a class method?

Reply:
Just cause it feels like this methods constructs an instance of the class, any reason not to? (I don't know much about OOP)